### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: ureport/urls.py [hunk 1]
```diff
@@ -19,6 +24,7 @@
     url(r'^manage/', include('ureport.countries.urls')),
     url(r'^manage/', include('ureport.assets.urls')),
     url(r'^users/', include('dash.users.urls')),
+    url(r'^jsi18n/$', 'django.views.i18n.javascript_catalog', js_info_dict)
```
Comment (nicpottier) [inline ureport/urls.py]: Have you tested this? Doesn't this need to be included in a template now?

Comment (norkans7) [inline ureport/urls.py]: sadly NO

**Comment by nicpottier** [comment]
Looks like dash and smartmin aren't included.. do you have those symlinked?

**Comment by norkans7** [comment]
Okay I had only dash symlink, I can add smartmin as well

**Comment by nicpottier** [comment]
Ya, what I look for in these is big patches of things being removed.. in this case all the smartmin translations.. which would have been sad. :(

**Comment by norkans7** [comment]
I was a bit confused before too, now this looks sane

**Comment by nicpottier** [comment]
Nice, looks good.

Ok next step is to upload the English .po file to Transifex. We can then get Maria to update the Spanish translations that are missing. 

Once she is done with that, then we download the .po files from Transifex, then do a ```fab compilemessages``` and push.

**Pull request merged**